tests/hikari/impl/test_event_manager_base.py

```diff
@@ -699,6 +700,25 @@ async def test(event):
         resolve_signature.assert_not_called()
         subscribe.assert_called_once_with(member_events.MemberCreateEvent, test, _nested=1)
 
+    def test_listen_when_multiple_params_provided_in_decorator(self, event_manager):
+        stack = contextlib.ExitStack()
+
+        subscribe = stack.enter_context(mock.patch.object(event_manager_base.EventManagerBase, "subscribe"))
+        resolve_signature = stack.enter_context(mock.patch.object(reflect, "resolve_signature"))
+
+        with stack:
+
+            @event_manager.listen(member_events.MemberCreateEvent, member_events.MemberDeleteEvent)
+            async def test(event):
+                ...
+
+        assert subscribe.call_count == 2
+        resolve_signature.assert_not_called()
+        subscribe.assert_has_calls(
+            subscribe(member_events.MemberCreateEvent, test, _nested=1),
+            subscribe(member_events.MemberDeleteEvent, test, _nested=1),
```

Comment:
```suggestion
            mock.call(member_events.MemberCreateEvent, test, _nested=1),
            mock.call(member_events.MemberDeleteEvent, test, _nested=1),
```

Although if this test passes, that is a new thing i learned about mock today. But would rather keep it consistent :)

Reply:
that was just me being dumb and forgetting to change it.